src/NUnitFramework/framework/Api/NUnitTestAssemblyRunner.cs

```diff
@@ -232,11 +232,7 @@ public void RunAsync(ITestListener listener, ITestFilter filter)
         /// <returns>True if the run completed, otherwise false</returns>
         public bool WaitForCompletion(int timeout)
         {
-#if !PORTABLE && !NETSTANDARD1_6
-            return _runComplete.WaitOne(timeout, false);
-#else
             return _runComplete.WaitOne(timeout);
-#endif
         }
```

Comment:
These two methods do the same thing on all platforms I believe, just Portable and Netstandard don't have the .WaitOne(int, bool) overload. I don't think the conditional is necessary here.